### PR TITLE
BST Validation in Python

### PR DESCRIPTION
**Pull request by harsha-sam**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes, I've read.

### Description
- Implemented Binary Search Tree Validation Check in python

### Checklist

- ✅  I've read the contribution guidelines.
- ✅  I've checked the issue list before deciding what to submit.
- ✅  I've edited the `README.md` and link to my code.

### CODE REVIEW

File: Python/ds/Binary-Search-Tree-Validation.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import typing
```
Comment (HarshCasper) [inline Python/ds/Binary-Search-Tree-Validation.py]: Add a beginner pitch about the Algorithm that you are implementing

File: Python/ds/Binary-Search-Tree-Validation.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    #    1     3    5       7
+    # This tree is a valid bst
+    root5 = createTree(0, [4, 2, None, 1, 5])
+    print(isValidBST(root5))
+    #           4
+    #         /
+    #        2
+    #     /    \
+    #    1      5
+    # This tree is invalid bst as node 5 in left sub tree of the root is greater than the root
```
Comment (HarshCasper) [inline Python/ds/Binary-Search-Tree-Validation.py]: Instead of Driver Code having multiple examples, prefer dynamic input.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
The Code looks good to me!

File: Python/ds/Binary-Search-Tree-Validation.py (hunk 1 above)
Comment (harsha-sam) [reply Python/ds/Binary-Search-Tree-Validation.py]: Sure.

File: Python/ds/Binary-Search-Tree-Validation.py (hunk 2 above)
Comment (harsha-sam) [reply Python/ds/Binary-Search-Tree-Validation.py]: I preferred multiple examples visualizing the tree only to make the code more understandable.

Comment (HarshCasper) [reply Python/ds/Binary-Search-Tree-Validation.py]: Its unsolicited. We only accept PRs with dynamic input. If not changed, we would not be able to merge the PR.

Comment (harsha-sam) [reply Python/ds/Binary-Search-Tree-Validation.py]: Done. @HarshCasper Can you re-verify again?

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**